modules/gapi/include/opencv2/gapi/fluid/gfluidkernel.hpp

```diff
@@ -104,12 +104,23 @@ struct GFluidParallelOutputRois
     std::vector<GFluidOutputRois> parallel_rois;
 };
 
+struct GFluidParallelFor
+{
+    std::function<void(std::size_t, std::function<void(std::size_t)>)> parallel_for;
```

Comment:
Please put a comment here what internal lambda argument (`std::size_t` is about)